.github/workflows/build.yml

```diff
@@ -251,11 +248,11 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Download sentry-native (Windows)
+      - name: Download sentry-native (win-x64)
```

Comment:
nitpick: consistency of `name`

(not related to this line, but to a line not changed in this PR)

above, in line: 145, we have
`- name: Download sentry-native (macos)`
but below, in line: 298
`- name: Download sentry-native (macOS)`

If you'd humor my ultimate nitpick, we could change the capitalization of the `name`, in line: 298 as well
```diff
-- name: Download sentry-native (macOS)
+- name: Download sentry-native (macos)
```